tools/autograph-monitor/monitor.go

```diff
@@ -67,37 +62,30 @@ func main() {
 	// prod or autograph dev code signing PKI roots and CA root
 	// certs defined in constants.go
 	conf.env = os.Getenv("AUTOGRAPH_ENV")
+	var err, depErr error
 	switch conf.env {
 	case "stage":
-		conf.rootHash = firefoxPkiStageRootHash
-		conf.truststore = x509.NewCertPool()
-		conf.truststore.AppendCertsFromPEM([]byte(firefoxPkiStageRoot))
-		conf.contentSignatureRootHash = firefoxPkiContentSignatureStageRootHash
-		conf.contentSignatureTruststore = x509.NewCertPool()
-		conf.contentSignatureTruststore.AppendCertsFromPEM([]byte(firefoxPkiContentSignatureStageRoot))
-		conf.depRootHash = ""
-		conf.depTruststore = nil
+		conf.truststore, conf.rootHashes, err = LoadCertsToTruststore(firefoxPkiStageRoots)
 	case "prod":
-		conf.rootHash = firefoxPkiProdRootHash
-		conf.truststore = x509.NewCertPool()
-		conf.truststore.AppendCertsFromPEM([]byte(firefoxPkiProdRoot))
-		conf.contentSignatureRootHash = firefoxPkiProdRootHash
-		conf.contentSignatureTruststore = conf.truststore
-		conf.depRootHash = firefoxPkiStageRootHash
-		conf.depTruststore = x509.NewCertPool()
-		conf.depTruststore.AppendCertsFromPEM([]byte(firefoxPkiStageRoot))
+		conf.truststore, conf.rootHashes, err = LoadCertsToTruststore(firefoxPkiProdRoots)
+		conf.depTruststore, conf.depRootHashes, depErr = LoadCertsToTruststore(firefoxPkiProdRoots)
 	default:
-		conf.rootHash = autographDevRootHash
-		conf.truststore = nil
-		conf.contentSignatureRootHash = autographDevRootHash
-		conf.contentSignatureTruststore = nil
-		conf.depRootHash = ""
-		conf.depTruststore = nil
+		_, conf.rootHashes, err = LoadCertsToTruststore(firefoxPkiDevRoots)
+	}
+
+	if err != nil {
+		err = fmt.Errorf("failed to load truststore root certificates: %w", err)
 	}
+	if depErr != nil {
+		depErr = fmt.Errorf("failed to load depTruststore root certificates: %w", depErr)
+	}
+	if err != nil || depErr != nil {
+		log.Fatalf("%s", errors.Join(err, depErr))
```

Comment:
This should allow us to know if either truststore is failing. Which could be useful in the future if we start using both for stage or dev for whatever reason.